### PR TITLE
add documentation and repository links to Cargo.toml for crates.io

### PR DESCRIPTION
**Pull request by vadixidav**
Commit message is self-explanatory.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,8 @@ name = "glium_text"
 version = "0.10.0"
 authors = ["Pierre Krieger <pierre.krieger1708@gmail.com>"]
 description = "Text drawing with glium and freetype"
+documentation = "https://docs.rs/glium_text/0.10.0/glium_text/"
```
Comment (tomaka) [inline Cargo.toml]: That should just be `https://docs.rs/glium_text`.

**Comment by vadixidav** [comment]
If you include the version information on each release, the user can pull up documentation for each relese by clicking the versions individually. Do you still want it to always point to the most recent docs?

**Comment by vadixidav** [comment]
@tomaka Nevermind, I looked into it and this doesn't seem to be the case. I will change it to that.

**Comment by tomaka** [comment]
Thanks

**Pull request merged**